.github/workflows/build_conda_linux.yml

```diff
@@ -183,11 +183,21 @@ jobs:
           ${CONDA_RUN_SMOKE} conda install \
             --yes \
             --quiet \
+            -c "${CONDA_LOCAL_CHANNEL}" \
             -c pytorch-"${CHANNEL}" \
             -c nvidia \
-            -c "${CONDA_LOCAL_CHANNEL}" \
-            distr::"${PACKAGE_NAME}" \
-            ${CONSTRAINTS}
+            -c defaults \
+            "${PACKAGE_NAME}" \
+            ${CONSTRAINTS} \
+            --override-channels
```

Comment:
```suggestion
            --override-channels \
            --strict-channel-priority
```

Reply:
Ok Thank you let me try it

Reply:
Looks like override works better, strict failed to resolve some of the installs:
https://github.com/pytorch/test-infra/actions/runs/7040631634/job/19161895215?pr=4770